AGE_65_PLUS.qmd

```diff
@@ -91,17 +99,47 @@ names(df) <-c("geoid" ,"region" ,"value", "moe")
 
 state_choropleth(df, 
                  title         = "POPULATION 65+ \n2021 1-YEAR ACS S0101_C01_030",
-                 legend        = "POPULATION")
+                 legend        = "POPULATION",
+                 num_colors = 4)
 
-ggplot(df, aes(y = value)) +
+x = ggplot(df, aes(y = value)) +
   geom_boxplot(color="red", fill="orange", alpha=1) + 
   scale_y_continuous(label=comma) +
   ggtitle("POPULATION DISTRIBUTION STATES\n WITH POPULATION 65+") +
   xlab("")+
 ylab("POPULATION")+
   theme(axis.text.x=element_blank(),
         axis.ticks.x=element_blank())
+ggplotly(x)
+
+```
+
+```{}
```

Comment:
Why are these empty lines here? Generally empty lines should not be in the document.